include/mata/nfa/strings.hh

```diff
@@ -113,6 +113,11 @@ private:
  */
 std::set<Word> get_shortest_words(const Nfa& nfa);
 
+/**
+ * @brief Get all the symbols accepted by @p nfa.
+ */
+std::set<mata::Symbol> get_one_symbol_words(const Nfa& nfa);
```

Comment:
Should we return a container of `Word` (which would all have only a single symbol, that is, they would be vectors of size one)?

And should we return `std::set` or `OrdVector`?

Reply:
Container of `Word` does not make sense here.

I am working with `std::set` in the function, so I am returning it, I am not sure if it makes sense to transform to `OrdVector`.

Reply:
I agree with you. I wanted to open the discussion since the functions does not return one symbol words: it returns a set of symbols of one symbol words accepted by the automaton. It is something different. Especially when we have a *word* reserved for a specific type.

Reply:
Then maybe I should rename the function?

Reply:
Maybe something like `get_accepted_symbols()`?

Reply:
I changed the name.

Reply:
I like the new name. I consider this conversation resolved.